pkg/controllers/management/auth/indexes.go

```diff
@@ -33,26 +40,33 @@ func rbRoleSubjectKeys(roleName string, subjects []v1.Subject) []string {
 	return keys
 }
 
+// indexByMembershipBindingOwner validate the object passed throught the arguments
+// and return the list of keys whose value is the equal to "memberhip-binding-owner".
```

Comment:
This comment doesn't sound quite right, it's used to index the "membership-binding-owner" field on resources.

(The code in `rbObjectKeys` is weird, given that labels are a map, so...how it could get more than one, I don't know?).

Reply:
> (The code in rbObjectKeys is weird, given that labels are a map, so...how it could get more than one, I don't know?).

To be honest I don't know what could be the scenario to get more than one, but since the check is done on the values of the map, this may not be unique, unlike the keys, so I preferred to leave the code as it was in the previous implementation.